crates/spirv-std/src/ray_tracing.rs

```diff
@@ -159,3 +159,473 @@ bitflags::bitflags! {
         const SKIP_AABBS = 512;
     }
 }
+
+/// A ray query type which is an opaque object representing a ray traversal.
+#[spirv(ray_query)]
+#[derive(Clone, Default)]
```

Comment:
Neither of these traits is applicable - you can't have a by-value `RayQuery`.
Also, it should either be unsized *or* used only through `Pin<&mut RayQuery>`, otherwise `mem::swap` may work.